configuration/themes.md

```diff
@@ -5,53 +5,76 @@ description: Zed UI and syntax themes explained
+{% endhint %}
 
-### Planned features
+_!!Under Construction!!_
 
-We'll talk more soon about our plans for exactly what will be themable, but likely _most_(but not all) of the Zed UI should be able to be accessed by the theme.
+### Roadmap
 
-Themes likely will start with only a small amount of customization (provide colors, recommend a font, etc) and grow more customizable over time.
+_Subject to extreme change. This is to provide some insight into our plans around themes, and the amount of work that needs to go into the system before user themes can be launched._
+
+_!!Under Construction!!_
+
+- [x] **Default Palette**: The set of colors that Zed pulls from to build it's default themes and uses for semantic colors. This is almost entirely internal facing, but eventually we will expose it for use in custon themes.
+- [ ] **Syntax Overrides**: The ability to override syntax colors from a theme. This will ship before public beta to allow us to correctly color the One theme family, which will continue to be the default for now.
+- [ ] **Theme 1.0 Specification**: The spec for a theme. This will include the ability to override syntax colors, as well as UI colors. We likely will define more features than we will build at the start, but this will allow us to add more features in the future without breaking existing themes.
+- [ ] **Port Existing Themes**: We will need to bring our existing themes up spec. Some themes will likely be removed or replaced at this point.
+- [ ] **Token Driven UI**: This is internal facing. Use standard design tokens to style every element in the UI. This is a pretty big step that won't have a lot of immediatly visible impact, but will allow us to build UI Overrides, and significantly improve the way we style every part of the app.
+- [ ] **Loading Themes**: The ability to load a theme from disk.
+- [ ] **UI Overrides**: Similar to syntax overrides, but for UI elements. This would, for example, allow you to change the color of the sidebar background or the label color of an active tab.
+
+---
 
-**Custom themes**
+## Known issues
 
-* There should be a list of curated themes available to download
-* Themes should be able to be loaded from a directory on the local machine
-* A format and builder should be available to help theme creators build themes for Zed.
+- **Low contrast**: The current contrast beween elements from theme to theme ranges from ok to very low. This will be addressed when we ship the tokens portion of our new system (see roadmap below)
+- **The colors of errors/warnings/something else look wrong**: This comes from our internal lack of (internal) color overrides. Our semantic colors are not separated from syntax/accent colors currently, which leads to leads to some themes having awkward error colors, etc. This will be tackled in the very near future.
```

Comment:
```suggestion
- **The colors of errors/warnings/something else look wrong**: This comes from our internal lack of color overrides. Our semantic colors are not separated from syntax/accent colors currently, which leads to some themes having awkward error colors, etc. This will be tackled in the very near future.
```
I don't think the `(internal)` is needed here.